werkzeug/contrib/limiter.py

```diff
@@ -31,7 +31,12 @@ class StreamLimitMiddleware(object):
     """
 
     def __init__(self, app, maximum_size=1024 * 1024 * 10):
-        warn(DeprecationWarning('This middleware is deprecated'))
+        warnings.warn(
+            'werkzeug.contrib.limiter is deprecated as of version 0.15 and'
```

Comment:
based on the doc-string this should be `0.9` instead of `0.15`

Reply:
If it started from 0.9. I think it is ok to remove it now.